merino-settings/src/lib.rs

```diff
@@ -90,6 +90,12 @@ pub struct Settings {
     /// off, the suggest request object should be logged, but the
     /// search query should be blank.
     pub log_full_request: bool,
+
+    /// Only used for integration tests.
+    // note `#[cfg(test)]` will cause a compile error as the
+    // `merino_test` proc_macro will not be able to find this
+    // field.
+    pub test_changes: Option<Vec<String>>,
```

Comment:
I'd think this should be nested under the Remote Settings configuration object, instead of at the top level, or at least mention that it is for Remote Settings. Also some documentation about what it is and how to do it would be warranted.